CMakeLists.txt

```diff
@@ -376,9 +376,9 @@ if(SDL_MIXER_X_STATIC AND NOT BUILD_AS_VB6_BINDING)
     if(NOT CMAKE_VERSION VERSION_LESS 3.0.2)
         set_target_properties(SDL2_mixer_ext_Static
             PROPERTIES
-            ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib/$<$<CONFIG:Debug>:>"
```

Comment:
Don't remove these lines, they are workaround for MSBuild to don't put binaries into sub-directories and put them at the same place as everywhere.